src/app/profile/duck/reducers.js

```diff
@@ -0,0 +1,38 @@
+import types from './types';
+
+const defaultState = {
+  uploadStatus: '',
+};
+
+const profileReducer = (state = defaultState, action) => {
+  switch (action.type) {
+    case types.SET_USER_PROFILE: {
+      const { profileData } = action;
+      return {
+        ...state,
+        profileData,
+      };
+    }
+
+    case types.VIEW_USER_PROFILE: {
+      const { viewData } = action;
+      return {
+        ...state,
+        viewData,
```

Comment:
How does viewData differ form profileData above?

Reply:
ViewData is data gotten when making an API call to view user data while profile data is that sent when making a patch request to update user profile...